zerver/tests/test_scheduled_messages.py

```diff
@@ -433,16 +434,11 @@ def test_editing_failed_send_scheduled_message(self) -> None:
         # successful and resets the `failed` and `failure_message` fields.
```

Comment:
Once we decide about what to do if the timestamp isn't updated on failed messages, it would be good to add a test for that here.

Reply:
Note added test coverage here for returning an error in the case that a failed scheduled message is being editing without an updated send timestamp.